uniflow/flow/transform/transform_summary_flow.py

```diff
@@ -0,0 +1,161 @@
+"""Paper Comparison Flow Module."""
+
+from typing import Any, Dict, Sequence
+
+from uniflow.constants import TRANSFORM
+from uniflow.flow.flow import Flow
+from uniflow.node import Node
+from uniflow.op.model.lm.model import LmModel
+from uniflow.op.model.model_op import ModelOp
+from uniflow.op.prompt import PromptTemplate
+from uniflow.op.prompt import Context
+
+from uniflow.op.basic.expand_op import ExpandOp
+from uniflow.op.basic.reduce_op import ReduceOp
+from uniflow.op.basic.group_op import GroupOp
+
+
+class GoogleModelFlow(Flow):
+    """Google Model Flow Class."""
+
+    def __init__(
+        self,
+        prompt_template: PromptTemplate,
+        model_config: Dict[str, Any],
+    ) -> None:
+        """Google Model Flow Constructor.
+
+        Args:
+            prompt_template (PromptTemplate): Guided prompt template.
+            model_config (Dict[str, Any]): Model config.
+        """
+        super().__init__()
+        self._model_op = ModelOp(
+            name="google_model_op",
+            model=LmModel(
+                prompt_template=prompt_template,
+                model_config=model_config,
+            ),
+        )
+
+        # Expand list of nodes to two or more nodes 
+        self.expand0_fn = lambda x: [[x[0][i]] for i in range(len(x[0]))]
+        self._expand0_op = ExpandOp(
+            name="google_model_exapnd0",
+            fn=self.expand0_fn
+        )
+
+        # Split into chunks
+        self.expand1_fn = lambda d: [[Context(context=item.strip())] for item in d[0].Context.split('\n') if item.strip()]
+
+        self._expand1_op = ExpandOp(
```

Comment:
name op with an index such as 0 or 1 is not a good practice and it can cause confusion during the review process. You should consider to rename with explicit expand functionality such as `self._nodes_expand_op` and `self._chunks_expand_op`